Index: src/main/java/com/fpt/jpos/utils/SecurityUtils.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/fpt/jpos/utils/SecurityUtils.java b/src/main/java/com/fpt/jpos/utils/SecurityUtils.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/fpt/jpos/utils/SecurityUtils.java	
@@ -0,0 +1,10 @@
+package com.fpt.jpos.utils;
+
+import com.fpt.jpos.pojo.Account;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class SecurityUtils {
+    public static Account getPrincipal() {
+        return (Account) (SecurityContextHolder.getContext()).getAuthentication().getPrincipal();
+    }
+}
